generated/thrift/rpc.thrift

```diff
@@ -49,11 +49,17 @@ service Node {
 	FetchRawBatchResult fetchRawBatch(1: FetchRawBatchRequest req) throws (1: Error err)
 	FetchBlocksResult fetchBlocks(1: FetchBlocksRequest req) throws (1: Error err)
 	FetchBlocksMetadataResult fetchBlocksMetadata(1: FetchBlocksMetadataRequest req) throws (1: Error err)
+	void truncateNamespace(1: TruncateNamespaceRequest req) throws (1: Error err)
+}
+
+struct TruncateNamespaceRequest {
+    1: required string ns
```

Comment:
`namespace` is a thrift keyword and thrift compiler doesn't like it unfortunatley :(